Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +979,44 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandBootstrapOption {
+    private let apply : ShorthandApply
+    
+    private init(apply : ShorthandApply) {
+        self.apply = apply
+    }
+    
+    /// Apply the contained option to the supplied ServerBootstrap
+    /// - Parameter serverBootstrap: bootstrap to apply this option to.
+    /// - Returns:the modified bootstrap (currently the same one mutated)
```

Comment:
```suggestion
    /// - Returns: the modified bootstrap (currently the same one mutated)
```

Reply:
Renamed to allowImmediateEndpointAddressReuse